syntax = "proto3";

message DashboardRequest {
    string service_name = 1;
    int64 timestamp = 2;
    string method = 3;
    int32 status = 4;
    string message = 5;
}

message DashboardResponse {
    int32 status = 1;
    string message = 2;
}

service DashboardService {
    rpc SendTraceInfo (stream DashboardRequest) returns (DashboardResponse);
}